src/style.css

```diff
@@ -1 +1,38 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
```

Comment:
You've fixed the width of `.stars` to 96px. Unless there's a specific requirement for this, it's better to let the content dictate the size of the container. This can help avoid overflow or accidental scroll bars.